Error when adding a module from module template in Windows

**Description:**
The following error is occurring when trying to add a module using a module template in Windows. This works fine in OS X.
```
[2019-10-09 09:35:37,361] SEVERE {b7a.log.crash} - Illegal character in opaque part at index 11: jar:file:C:\Users\wso2\.ballerina\balo_cache\wso2\hello_world_service\1.0.0\hello_world_service-2019r3-any-1.0.0.balo 
java.lang.IllegalArgumentException: Illegal character in opaque part at index 11: jar:file:C:\Users\wso2\.ballerina\balo_cache\wso2\hello_world_service\1.0.0\hello_world_service-2019r3-any-1.0.0.balo
	at java.net.URI.create(URI.java:852)
	at org.ballerinalang.packerina.cmd.AddCommand.applyBaloTemplate(AddCommand.java:265)
	at org.ballerinalang.packerina.cmd.AddCommand.createModule(AddCommand.java:249)
	at org.ballerinalang.packerina.cmd.AddCommand.execute(AddCommand.java:198)
	at java.util.Optional.ifPresent(Optional.java:159)
	at org.ballerinalang.tool.Main.main(Main.java:59)
Caused by: java.net.URISyntaxException: Illegal character in opaque part at index 11: jar:file:C:\Users\wso2\.ballerina\balo_cache\wso2\hello_world_service\1.0.0\hello_world_service-2019r3-any-1.0.0.balo
	at java.net.URI$Parser.fail(URI.java:2848)
	at java.net.URI$Parser.checkChars(URI.java:3021)
	at java.net.URI$Parser.parse(URI.java:3058)
	at java.net.URI.<init>(URI.java:588)
	at java.net.URI.create(URI.java:850)
	... 5 more
```

**Affected Versions:**
Ballerina 1.0.1

**OS, DB, other environment details and versions:**
Windows

**Suggested Assignees (optional):**
@hevayo 
@hemikak

--- a/cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/AddCommand.java
+++ b/cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/AddCommand.java
@@ -262,7 +262,7 @@ public class AddCommand implements BLauncherCmd {
         if (baloTemplate != null) {
             String moduleName = getModuleName(baloTemplate);
 
-            URI zipURI = URI.create("jar:file:" + baloTemplate.toString());
+            URI zipURI = URI.create("jar:" + baloTemplate.toUri().toString());
             try (FileSystem zipfs = FileSystems.newFileSystem(zipURI, new HashMap<>())) {
                 // Copy sources
                 Path srcDir = zipfs.getPath("/src").resolve(moduleName);
@@ -403,7 +403,7 @@ public class AddCommand implements BLauncherCmd {
     }
 
     private Module getModuleToml(Path baloPath) {
-        URI zipURI = URI.create("jar:file:" + baloPath.toString());
+        URI zipURI = URI.create("jar:" + baloPath.toUri().toString());
         try (FileSystem zipfs = FileSystems.newFileSystem(zipURI, new HashMap<>())) {
             Path metaDataToml = zipfs.getPath("metadata", "MODULE.toml");
             // We do a string comparison to be efficient.
@@ -416,7 +416,7 @@ public class AddCommand implements BLauncherCmd {
     }
 
     private boolean isTemplateBalo(Path baloPath) {
-        URI zipURI = URI.create("jar:file:" + baloPath.toString());
+        URI zipURI = URI.create("jar:" + baloPath.toUri().toString());
         try (FileSystem zipfs = FileSystems.newFileSystem(zipURI, new HashMap<>())) {
             Path metaDataToml = zipfs.getPath("metadata", "MODULE.toml");
             // We do a string comparison to be efficient.

